mkdocs.yml

```
@@ -67,18 +67,16 @@ nav:
   - Prepare your metadata: TSVTools.md
   - Prepare your imaging data:
     - Introduction: Preprocessing/Introduction.md
-    - t1-linear pipeline: Preprocessing/T1_Linear.md
-    - t1-extensive pipeline: Preprocessing/T1_Extensive.md
-    - extract tensors: Preprocessing/Extract.md
-    - quality check: Preprocessing/QualityCheck.md
-    - generate synthetic data sets: Generate.md
+    - Extract tensors: Preprocessing/Extract.md
+    - Quality check: Preprocessing/QualityCheck.md
+    - Generate synthetic data sets: Preprocessing/Generate.md
   - Hyperparameters exploration: RandomSearch.md
   - Train deep learning networks:
     - Introduction: Train/Introduction.md
     - JSON defined: Train/Retrain.md
     - Resume: Train/Resume.md
     - Implementation details: Train/Details.md
```

I find training subsection titles not clear, I suggest:
```suggestion
  - Train deep learning networks:
    - Train: Train/Introduction.md
    - Retrain from a JSON: Train/Retrain.md
    - Resume training: Train/Resume.md
    - Implementation details: Train/Details.md
```

---

I agree, maybe we can discuss it with @14thibea before committing these changes and also discuss about the `JSON defined` section that should be removed.